### ISSUE TITLE
Error while playing the music on "on device"

### ISSUE THREAD

**Issue by Tanujairam-TG** [fast4x/RiMusic#205]
### Steps to reproduce the bug

1.Go to library
2. Click "on device"
3. Click any song 

### Expected behavior

This should play the song

### Actual behavior

Showing the error 
![Screenshot_2023-12-10-15-35-25-946](https://github.com/fast4x/RiMusic/assets/88611197/ce60ce12-a0bc-4bba-b6a4-9590288a4ee4)

### RiMusic version

0.6.12

### Android version

Android 10

**Comment by fast4x** [comment]
Yeah, it’s a known bug.

**Comment by Tanujairam-TG** [comment]
Then how to fix

**Comment by 25huizengek1** [comment]
Allow me to save you guys some time, I've encountered this problem when I added Local music : the problem is that the `ExtractorsFactory` currently only supports Matroska and fragmented mp4, so when in this case a mp3 file is trying to play, the player doesn't know how to handle this file type (parsing of the container is unsupported). 

To fix this:
https://github.com/fast4x/RiMusic/blob/50fa4c0ddd5fb5e5f5c5875c86351be9fe49ea87/app/src/main/kotlin/it/vfsfitvnm/vimusic/service/PlayerService.kt#L1106

This function should return `DefaultExtractorsFactory`, a 'shorthand' from Exoplayer that supports more containers. It's a good idea to handle some errors too: for example this error showing in the screenshot, and when a local song is added to a playlist and then got deleted, it should show a nice, localized error.

I've fixed this in #209
